### PR TITLE
make monkeys better

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
1. [X] give monkeys doorbumpopener
2. [X] let monkeys strip people
3. [X] make syndie monkeys traitor

too hard to give them objective of helping whoever bought so didnt do it

also why does normal monkey have mouse tag tf

depends on https://github.com/space-wizards/RobustToolbox/pull/4290

## Why / Balance
1. they can already click and press e on doors to open them so why not
2. trained monkeys are famous for stealing shit so it makes sense, also gives syndie monkey a bit of utility since it can steal cap id for you
   might move to syndie monkeys only incase it would cause too much shittery
3. makes sense for them to show up also admin stuff(?) also turrets and grenade penguins

## Technical details
`TraitorRuleSystem.MakeTraitor` now has `giveUplink` and `giveObjectives` options which default to true, admin traitor smite uses presence of HumanoidComponent for these.
added `AutoTraitorComponent` which makes the entity a traitor either immediately if it has a mind or when a mind is added

## Media
codewords shown in character info of evil monke

listed in traitor round end text

evil

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:cl:
- tweak: Monkeys can now bump doors open and strip mobs like people can.

### CODE REVIEW

**Comment by deltanedas** [comment]
for some reason on monkey it doesnt say briefing in char menu even tho its in vv...

**Comment by Jezithyr** [comment]
This is pretty funny and peak SpessStation, Looks pretty good to me!

File: Content.Server/Traitor/Systems/AutoTraitorSystem.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AutoTraitorComponent, ComponentStartup>(OnStartup);
+        SubscribeLocalEvent<AutoTraitorComponent, MapInitEvent>(OnMapInit);
+        SubscribeLocalEvent<AutoTraitorComponent, MindAddedMessage>(OnMindAdded);
+    }
+
+    // workaround for #19409
+    private void OnStartup(EntityUid uid, AutoTraitorComponent comp, ComponentStartup args)
```
Comment (metalgearsloth) [inline Content.Server/Traitor/Systems/AutoTraitorSystem.cs:28]: I will just code this you noob.

Comment (deltanedas) [reply Content.Server/Traitor/Systems/AutoTraitorSystem.cs:28]: ok troller say when it is :trollface:

Comment (deltanedas) [reply Content.Server/Traitor/Systems/AutoTraitorSystem.cs:27]: @metalgearsloth :trollface:

Comment (metalgearsloth) [reply Content.Server/Traitor/Systems/AutoTraitorSystem.cs:27]: I got like 10 layers deep of soap but still working on it.

Comment (deltanedas) [reply Content.Server/Traitor/Systems/AutoTraitorSystem.cs:27]: pro

**Comment by deltanedas** [comment]
@metalgearsloth mapinit works now nice

**Pull request merged**